process/block/shardblock_test.go

```diff
@@ -3085,6 +3085,154 @@ func TestShardProcessor_CreateAndProcessCrossMiniBlocksDstMe(t *testing.T) {
 	assert.Equal(t, noOfTxs, uint32(0))
 }
 
+func TestShardProcessor_CreateAndProcessCrossMiniBlocksDstMeShouldErr(t *testing.T) {
+	t.Parallel()
+	haveTimeReturnsBool := func() bool {
+		return true
+	}
+	tdp := mock.NewPoolsHolderFake()
+	txHash := []byte(nil)
+	tdp.Transactions().AddData(txHash, &transaction.Transaction{}, process.ShardCacherIdentifier(1, 0))
+
+	sp, _ := blproc.NewShardProcessor(
+		&mock.ServiceContainerMock{},
+		tdp,
+		&mock.ChainStorerMock{},
+		&mock.HasherStub{},
+		&mock.MarshalizerMock{},
+		&mock.AccountsStub{},
+		mock.NewMultiShardsCoordinatorMock(3),
+		&mock.ForkDetectorMock{},
+		&mock.BlocksTrackerMock{},
+		createGenesisBlocks(mock.NewMultiShardsCoordinatorMock(3)),
+		false,
```

Comment:
here you should have true, createandprocesscrossminiblocksdstme verifies metablocks according to last notarized. This input can be delete from NewShardProcessor, as it is not more possible to run without metachain.

Reply:
metaChainActive is set true now and after that verify if ErrWrongTypeAssertion is returned if in startHeader parameter i put a wrong type of block.